### DISCUSSION TITLE
It has been a while since a release....

### DISCUSSION THREAD

**Discussion by Audiojunkie** [asb2m10/dexed#397] [General]
I know everyone is busy, but I'm wondering if maybe an interim release, for example 0.9.6.5, could be released with the GUI fixes.  It has been nearly 2 and a half years since a release, and I'd really love to be able to use this plugin.  I use a 4k monitor on a laptop, and the HiDPI scaling is still broken.  Thanks!

**Comment by asb2m10** [comment]
Yes, I'm planning to release 0.9.7 in December since there is some important fixes and implementation. I need more feedback on the latest build. Let me know if you do see any issues with these ones: https://github.com/asb2m10/dexed/actions/runs/6518805707

**Reply by Audiojunkie** [reply]
This is fantastic news! Thank you!!! I look forward to it! 😎👍🏼
